scripts/prepublish.js

```diff
@@ -9,11 +9,15 @@ shell.echo(chalk.bold(`${packageJson.name}@${packageJson.version}`));
 shell.echo(chalk.gray('\n=> Clean dist.'));
 shell.rm('-rf', 'dist');
 
+const pwd = shell.pwd();
+shell.echo(`pwd = ${pwd}`);
+shell.cd(path.join(__dirname, '..'));
```

Comment:
Can you explain this a bit more? Doesn't babel end up looking for the preset in root if it doesn't find it in the package anyway (w/ standard `node_modules` resolution algorithm).